sqlcop/checks/cross_join.py

```diff
@@ -39,7 +40,8 @@ def __call__(self, stmt):
                     tok.match(sqlparse.tokens.Keyword.DML, 'SELECT') or
                     tok.match(sqlparse.tokens.Keyword, 'INNER JOIN') or
                     tok.match(sqlparse.tokens.Keyword, 'GROUP') or
-                    tok.match(sqlparse.tokens.Keyword, 'LEFT OUTER JOIN')):
+                    tok.match(sqlparse.tokens.Keyword, 'LEFT OUTER JOIN') or
+                    (isinstance(tok, sqlparse.sql.Identifier) and tok.value == 'LIMIT')):
```

Comment:
why does this pattern have to be so different?

(I'm just curious because I don't really know much about sqlcop -- also, hi Jill!)

Reply:
Hi Wyatt!!! 

Jill and I have a bet right now, I'm of the opinion it shouldn't be any different than the others!
`LIMIT` is a keyword, (i've just been poking around in `sqlparser` code)

Reply:
Matt's Evolve PR is failing because of this check. We are of the opinion that his query (similar to the query in the test) is being incorrectly flagged as cross join. Hence this fix.

Matt has his personal opinion 😉 

The Mark will decide!

Reply:
(hi Matt :smile: -- I ended up looking through the sqlparser code too haha)

Should `ORDER BY` be in this list too? If I follow the code right, I would think it would signal the `FROM` is over, and a two-element list of things to order by might count as a similar false-positive `IdentifierList`?

If it's never been a problem then /shrug maybe

Reply:
@wcarss https://github.com/freshbooks/sqlcop/blob/master/sqlcop/checks/__init__.py#L9
lol. 😞 

I'll look into why this was done tomorrow. If it isn't necessary, I'll remove it, or perhaps there's another, better way it can be done.
I'll double check if `ORDER_BY` is in the list too.

Reply:
Seems like `ORDER_BY` is not in `sqlparse.keywords.KEYWORDS`, but ORDER is in `sqlalchemy.dialects.mysql.base.RESERVED_WORDS`